recipe/build_osx_arm64.sh

```diff
@@ -0,0 +1,28 @@
+export VERBOSE=1
+
+echo "***************************************"
+echo "***************************************"
+echo "***************************************"
+echo "CMAKE_ARGS=${CMAKE_ARGS}"
+echo "PYTHON=${PYTHON}"
+echo "PREFIX=${PREFIX}"
+echo "BUILD_PREFIX=${BUILD_PREFIX}"
+echo "CONDA_PREFIX=${CONDA_PREFIX}"
+echo "CFLAGS=${CFLAGS}"
+echo "CXXFLAGS=${CXXFLAGS}"
+
+export CMAKE_PREFIX_PATH=${PREFIX}
+
+# Filter CMAKE_PREFIX_PATH out of CMAKE_ARGS because scikit-build needs to set it
+export SKBUILD_CONFIGURE_OPTIONS=$(echo $CMAKE_ARGS | sed -e 's/\-DCMAKE_INSTALL_PREFIX\=[^ ]* //g')
```

Comment:
Why not just modify the `CMAKE_ARGS` directly in this script? There's nothing being built that uses them other than freud anyway, right?